SlicerRadiomics/SlicerRadiomics.py

```diff
@@ -1,10 +1,11 @@
 import os
 import unittest
-import vtk, qt, ctk, slicer
+import vtk, qt, ctk, slicer, logging
```

Comment:
Let me guess - probably because I didn't pay enough attention! ;-)

Thanks, I will fix this among other things that need to be cleaned up. I will let you know when I consider it is ready for review/merge.